head/firmware/can_task.cpp

```diff
@@ -153,63 +155,108 @@ static motor_class::Motor motor_left{
+
+static auto device_info_handler_left =
+    can_device_info::DeviceInfoHandler(message_writer_left, 0);
+static auto device_info_dispatch_target_left =
+    DispatchParseTarget<decltype(device_info_handler_left),
+                        can_messages::DeviceInfoRequest>{device_info_handler_left};
+
+
+static auto motor_dispatch_target_right = DispatchParseTarget<
+    decltype(can_motor_handler_right), can_messages::SetupRequest,
     can_messages::StopRequest, can_messages::EnableMotorRequest,
     can_messages::DisableMotorRequest,
     can_messages::GetMotionConstraintsRequest,
     can_messages::SetMotionConstraints, can_messages::WriteMotorDriverRegister,
-    can_messages::ReadMotorDriverRegister>{can_motor_handler};
+    can_messages::ReadMotorDriverRegister>{can_motor_handler_right};
 
-static auto motor_dispatch_target2 = DispatchParseTarget<
-    decltype(can_motor_handler2), can_messages::SetupRequest,
+static auto motor_dispatch_target_left = DispatchParseTarget<
+    decltype(can_motor_handler_left), can_messages::SetupRequest,
     can_messages::StopRequest, can_messages::EnableMotorRequest,
     can_messages::DisableMotorRequest,
     can_messages::GetMotionConstraintsRequest,
     can_messages::SetMotionConstraints, can_messages::WriteMotorDriverRegister,
-    can_messages::ReadMotorDriverRegister>{can_motor_handler2};
+    can_messages::ReadMotorDriverRegister>{can_motor_handler_left};
 
-static auto motion_group_dispatch_target = DispatchParseTarget<
-    decltype(can_move_group_handler), can_messages::AddLinearMoveRequest,
+static auto motion_group_dispatch_target_right = DispatchParseTarget<
+    decltype(can_move_group_handler_right), can_messages::AddLinearMoveRequest,
     can_messages::GetMoveGroupRequest, can_messages::ClearAllMoveGroupsRequest>{
-    can_move_group_handler};
+    can_move_group_handler_right};
 
-static auto motion_group_executor_dispatch_target =
-    DispatchParseTarget<decltype(can_move_group_executor_handler),
+static auto motion_group_dispatch_target_left = DispatchParseTarget<
+    decltype(can_move_group_handler_left), can_messages::AddLinearMoveRequest,
+    can_messages::GetMoveGroupRequest, can_messages::ClearAllMoveGroupsRequest>{
+    can_move_group_handler_left};
+
+static auto motion_group_executor_dispatch_target_right =
+    DispatchParseTarget<decltype(can_move_group_executor_handler_right),
                         can_messages::ExecuteMoveGroupRequest>{
-        can_move_group_executor_handler};
+        can_move_group_executor_handler_right};
 
-static auto motion_group_executor_dispatch_target2 =
-    DispatchParseTarget<decltype(can_move_group_executor_handler2),
+static auto motion_group_executor_dispatch_target_left =
+    DispatchParseTarget<decltype(can_move_group_executor_handler_left),
                         can_messages::ExecuteMoveGroupRequest>{
-        can_move_group_executor_handler2};
+        can_move_group_executor_handler_left};
+
+struct CheckForNodeId {
```

Comment:
Can we drop a comment here that messages to `head 0x50` are really "head-both"? might come up in the future if we have a "control plane" concept or something